src/params.rs

```diff
@@ -259,6 +265,40 @@ impl<S: AsRef<str> + std::fmt::Debug + PartialEq<S> + From<&'static str>> TryFro
 	}
 }
 
+/// Enforcement strictness for file sandbox
+#[cfg(target_os = "linux")]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum FileSandboxMode {
+	/// Do not enable Landlock's path-based isolation features.
+	None,
+	/// Enable Landlock's path-based isolation features on a best-effort basis.
+	BestEffort,
+	/// Enforce Landlock rulesets strictly.
+	Strict,
+}
```

Comment:
- Don't mention Landlock here to be more universal
- Rename `BestEffort` to `Default` or `Normal`